spec/fixtures/imports/logs/0b4a68df-30cc-474a-93c0-a56ce8fdad3b.xml

```diff
@@ -19,7 +19,7 @@
     <FORM>123456</FORM>
     <Landlord source-value="2">2 Local Authority</Landlord>
     <Group>
-      <_1cmangroupcode>123</_1cmangroupcode>
+      <_1cmangroupcode>0123</_1cmangroupcode>
```

Comment:
Ids in the old system are only ever padded to three digits so I wouldn't expect to see anything like this, but this should still work as a test

Reply:
I think that's fine. The test here is that we accept _any_ value as a string without any modifications. We're not going to make any assumptions about what that field can look like.